scripts/landsat.py

```diff
@@ -1,17 +1,20 @@
+from datetime import datetime
+
 from watch_helpers import post_stac_items_from_server
 
-host_url = 'https://landsatlook.usgs.gov/stac-server/collections/landsat-c2l1/items'
+min_date = datetime(2013, 1, 1)  # Arbitrarily chosen
```

Comment:
@matt-bernstein, any thoughts on a minimum ingest date for Landsat? 

I'm waiting to see how long it takes to ingest, so assuming the last 8 years of data doesn't take more than an hour or so to ingest, there shouldn't be much of a limitation on our end